FirebaseAppDistribution/Sources/FIRAppDistributionAppDelegateInterceptor.m

```diff
@@ -100,7 +113,8 @@ - (void)showUIAlert:(UIAlertController *)alertController {
 - (BOOL)application:(UIApplication *)application
             openURL:(NSURL *)URL
             options:(NSDictionary<NSString *, id> *)options {
-  self.registrationFlowCompletion(nil);
+  FIRFADDebugLog(@"Continuing registration flow: %@", [self registrationFlowCompletion]);
+  if (self.registrationFlowCompletion) self.registrationFlowCompletion(nil);
```

Comment:
When is this thing nil? Should we instead capture this as an error?

Reply:
If you open the app via the URL scheme without going through the registration flow, this can be nil

This definitely can happen in non-error cases, which is why I just decided to log it instead.

Reply:
👍  Sounds good!